src/dhtproto/client/internal/NodeHashRanges.d

```diff
@@ -198,6 +201,14 @@ private class NodeHashRangesBase
             }
         }
 
+        bool sortPred ( NodeHashRange e1, NodeHashRange e2 )
+        {
+            assert(e1.order != e2.order);
```

Comment:
Too early for `verify`?

Reply:
Mmm... Is it released yet?

Reply:
Yes, in ocean `v2.10.0`/`v3.4.0`.

Reply:
Cool, I can update.

Reply:
> Cool, I can update.

I assume you figured out that you can't update, since we don't support this ocean yet?

Reply:
Oh I forgot about this. What do you mean we don't support it yet?

Reply:
If dhtproto/swarm is using ocean prior to `v3.4.0`, `verify` is not yet available.

Reply:
I could update the submodule in this (semver-free) branch.

Reply:
I won't bother. It introduces zillions of deprecation warnings. Something for a separate PR.